### DISCUSSION TITLE
IFTTT Support

### DISCUSSION THREAD

**Discussion by ghost** [bluesky-social/atproto#2049] [General Discussion]
Do you know of any plans to have ifttt integration?

https://ifttt.com/docs

**Comment by tcifelli** [comment]
+1 to this, I've historically been able to automate discord integration for twitter feeds with IFTTT and was looking to do the same with bluesky

**Reply by ghost** [reply]
you can use bluesky > to twitter without ifttt, because bluesky has an rss feed and if you use rss to discord it works as an immediate alternative,

but i also again would like ifttt for a lot of other uses with bluesky, it would also make for managing brand accounts a lot easier

**Comment by bnewbold** [comment]
Is that something that needs support on the bsky side? My memory/understanding of IFTTT is that it is a a platform for stitching together APIs, and I would assume it works fine with the bsky API (similarly to twitter, mastodon, etc).

**Reply by ghost** [reply]
best case scenario is bluesky makes an ifttt app (or a community member does, and then gives a official bluesky member account access so it can become an official app) so it gets wide spread support  (which would also mean any account anywhere on the protocol will be able to use it. 


![image](https://github.com/bluesky-social/atproto/assets/66647238/521b2e9f-6e1c-4c99-a9b2-44b9fafa9a4a)
email i had with an ifttt staff earlier

**Comment by ghost** [comment]
in the process- adding ifttt especially if it is first party,  would also make it easier to be able to program in with the protocol, being able to export  as an option, so users could auto backup/sync to ifttt to whatever third party of choice automatically,

**Comment by ylevch** [comment]
Both IFTTT and BlueSky work with RSS.
You can just add "/rss" to the URL of an account you're interested in (e.g. https://bsky.app/profile/bsky.app/rss), and then use IFTTT RSS integrations

**Reply by ghost** [reply]
This only works one direction tho, ifttt could offer more integration than just posting, scheduled or automated account export, updating pfp or display name n addition to posting

**Comment by MasterJ93** [comment]
If I have the time, I could look into it and whip something up.

(It'll probably be written in Swift, though, so if IFTTT doesn't support that, then I'll try and see if I can get it to work via Python or another language...)

**Comment by aendra-rininsland** [comment]
I looked into this a bit yesterday because I've had a similar idea for awhile, and I think it's a bit blocked by the lack of OAuth support; I'm not sure if [this bit I've screenshot below](https://ifttt.com/docs/api_reference) simply means "don't hardcode developer keys", but I'm reading it as "this is blocked by #1154".

![Screenshot_20240304-091419](https://github.com/bluesky-social/atproto/assets/185041/9ee5ed35-7e0a-43af-9de0-a82bf4136649)

**Comment by RayneYoruka** [comment]
Is there any update on this? Curious if there is any short or way to get posts from (anywhere) in to Bsky

**Reply by MasterJ93** [reply]
Now that OAuth is, for the most part, available, someone might end up making it compatible with IFTTT.

**Comment by corywright** [comment]
I'd very much like to have the ability to post to Bluesky via an IFTTT applet as we've been doing with Twitter/X for many years. In order to move our users to Bluesky we need to be able to offer them what they are currently getting on X.

**Comment by therourke** [comment]
I am also holding out for Bluesky support on IFTTT. It would be fantastic for posting to an account or a feed or updating a list, and all these things should be possible.

**Comment by RayneYoruka** [comment]
Now with the amount of users.. I agree

**Comment by msfriese** [comment]
It seems IFTTT can post to Buffer. Buffer can post to BlueSky. Maybe this can be a workaround while IFTTT cannot post straight to BlueSky?

*Buffer ist actually pretty good!

**Comment by PhilC813** [comment]
For some reason, this discussion can't be found with the search bar of the repo or the organization as a whole.

Should we open an issue instead? It's more likely that the BlueSky team will review the request if so.

**Comment by rghedin** [comment]
Any news on this?
